### PR TITLE
TODOs in "app/api/..."  Got rid of db calls

### PR DESCRIPTION
**Pull request by tsahil01**
Got rid of db call by using getServerSession()

### CODE REVIEW

**Comment by SujithThirumalaisamy** [comment]
LGTM!

**Comment by tsahil01** [comment]
> LGTM!

Can we merge it?

File: app/api/streams/downvote/route.ts [hunk 1]
```diff
@@ -26,11 +22,11 @@ export async function POST(req: NextRequest) {
     }
 
     try {
-        const data = UpvoteSchema.parse(await req.json());
+        const data = UpvoteSchema.parse(await req.json()); // data contains both streamId and userId
```
Comment (SujithThirumalaisamy) [inline app/api/streams/downvote/route.ts:25]: can we use `safeParse` and return a error status with the appropriate err message? Always try to use `safeParse` and return the error on the apis

Comment (tsahil01) [reply app/api/streams/downvote/route.ts:25]: Working on it!

Comment (tsahil01) [reply app/api/streams/downvote/route.ts]: Hey @SujithThirumalaisamy 
I added safeParse and improve error handling. 
Let me know for any other issues!

**Comment by SujithThirumalaisamy** [comment]
Closing on #76

**Pull request closed without merging**